tests/boardfarm_plugins/boardfarm_sample_plugin/tests/initial_ap_config.py

```diff
@@ -0,0 +1,42 @@
+
+import boardfarm
+
+from boardfarm.tests import bft_base_test
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    '''
+    PrplMesh sample test case, no actual testing, it just passes
+    '''
+
+    def runTest(self):
+        '''
+        Main test logic must be implemented here.
+        '''
+        print('{}: Running...'.format(self.__class__.__name__))
+
+    def skipTest(self, msg: str = ""):
+        '''
+        Method invoked if DUT's consoles don't respond,
+        generally considering device as 'dead'(in normal workflow).
+        Also, it can be used to provide info about possible failure reasons.
+        '''
+        print('{}: Skipping...'.format(self.__class__.__name__))
+        raise boardfarm.exceptions.SkipTest(msg)
+
+    def recover(self):
+        '''
+        Method invoked if test fails with an unexpected exception type.
+        According to boardfarm sources, it is considered as deprecated,
```

Comment:
If it is deprecated, what is the "new" way of recovering from a hard failure?

Reply:
> If it is deprecated, what is the "new" way of recovering from a hard failure?

There's no "new" way for current version(v1.1.1), warning message states that it will be provided in future versions.